pylana/aggregation.py

```diff
@@ -0,0 +1,215 @@
+"""
+aggregation api requests
+"""
+
+import pandas as pd
+
+from typing import Union
+from pylana.api import API
+from pylana.utils_aggregation import create_metric, create_grouping
+
+
+class AggregationAPI(API):
+
+    def extract_chart_values(self, jsn):
+        """Cast json into data frame and remove artifacts columns.
+
+        No error is raised in case we don't find artifact columns.
+        """
+        return pd.DataFrame(jsn['chartValues']) \
+            .drop(columns=['$type'], errors='ignore')
+
+    def normalise_chart_values(self, df, json_col):
+        df_aux = df \
+            .explode(json_col) \
+            .drop(columns=['$type'], errors='ignore') \
+            .reset_index(drop=True)
+        return pd.concat(
+            [
+                pd.json_normalize(df_aux.loc[:, json_col]).drop(columns=[
+                    '$type'], errors='ignore'),
+                df_aux.drop(columns=[json_col])
+            ], axis=1
+        )
```

Comment:
These methods  do not use any other method or attribute of the class. We either can make the method [static](https://docs.python.org/3/library/functions.html#staticmethod). Or we can remove it from the class, since the api user has no use for them.